apps/client/src/components/Questions.js

```diff
@@ -10,6 +10,8 @@ const Questions = ({ checker, topic: { slug } }) => {
   const sessionContext = useContext(SessionContext);
   const [editQuestion, setEditQuestion] = useState(false);
   const { questionIndex, finishedQuestions } = sessionContext[slug];
+  // Styling to overwrite the line between the Items
+  const activeStyle = { marginTop: -1, borderColor: "white" };
```

Comment:
We have this already on chappie 2.0 right?

Reply:
Yes. But since I've made it conditional I have moved it up. So it's not duplicated, but moved